recipes/treematching/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "treematching" %}
+{% set version = "0.1.20210513" %}
```

Mark this more clearly as a pre-release.
```suggestion
{% set version = "0.0.1a20210513" %}
```